keyboards/automata02/alisaie/info.json

```
@@ -0,0 +1,104 @@
+{
+    "manufacturer": "Automata02",
+    "keyboard_name": "Alisaie",
+    "maintainer": "qmk",
+    "bootloader": "atmel-dfu",
+    "bootmagic": {
+        "enabled": true
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["B6", "F0", "F1", "F4", "F5", "B5", "B7", "D3", "D2", "D1", "D0", "B3", "B2", "B0"],
+        "rows": ["E6", "F6", "D5", "D4", "B1"]
+    },
+    "mouse_key": {
+        "enabled": false
+    },
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x2012",
+        "vid": "0xBABE"
+    },
+    "layouts": {
+        "LAYOUT_split_bs_rshift": {
```

This change will also need to be reflected in the keymaps.
```suggestion
        "LAYOUT": {
```

---

Has been updated. Physically board supports split and non split so I would have liked to kept it but I can revert at a later date when and if I make a second layout.